src/service/ui/src/server.ts

```diff
@@ -2,6 +2,7 @@ import * as bodyParser from 'body-parser'
 import * as compression from 'compression'
 import * as express from 'express'
 import * as session from 'express-session'
+import * as cache from 'express-cache-response'
```

Comment:
For future, imported variables should not be callable, as that's not forward-compatible with the ES Modules spec. We've made a couple of exceptions so far, e.g. `express`, should also note this exception and try and avoid them in future.